README.md

```diff
@@ -79,23 +103,64 @@ $ composer create-project roots/bedrock site
 
 ## Remote server setup (staging/production)
 
-For remote servers, installing Ansible locally is an additional requirement. See the [docs](https://roots.io/trellis/docs/remote-server-setup/#requirements) for more information.
-
-A base Ubuntu 18.04 (Bionic) server is required for setting up remote servers. OS X users must have [passlib](http://pythonhosted.org/passlib/install.html#installation-instructions) installed.
+A base Ubuntu 18.04 (Bionic) server is required for setting up remote servers.
 
 1. Configure your WordPress sites in `group_vars/<environment>/wordpress_sites.yml` and in `group_vars/<environment>/vault.yml` (see the [Vault docs](https://roots.io/trellis/docs/vault/) for how to encrypt files containing passwords)
 2. Add your server IP/hostnames to `hosts/<environment>`
 3. Specify public SSH keys for `users` in `group_vars/all/users.yml` (see the [SSH Keys docs](https://roots.io/trellis/docs/ssh-keys/))
-4. Run `ansible-playbook server.yml -e env=<environment>` to provision the server
+
+### Using trellis-cli
+
+Provision the server:
+```bash
+$ trellis galaxy install
```

Comment:
We should mention `trellis init` so that users have correct version of ansible installed

Reply:
Updated